src/test/functional/features/joint-hub-page.feature

```diff
@@ -57,3 +57,12 @@ Feature: Joint hub page
     And I login with applicant "2"
     And the page should include "The hearing will take place at Birmingham Civil and Family Justice Centre on 29 September 2013 at 3:30PM."
     And the page should include "You can view and download your 'certificate of entitlement for a conditional order'."
+
+    Given I set the case state to "AwaitingClarification"
+    When I click "Sign out"
+    And I login with applicant "1"
+    And the page should include "Either you or your husband can provide the information requested by the court"
```

Comment:
should this be then

Reply:
ngl i barely check the structure, as long as the rest does what it does as cucumber gets the function anyways.

Reply:
ill still update it tho